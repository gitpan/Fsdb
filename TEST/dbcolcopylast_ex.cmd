prog='dbcolcopylast'
args='experiment mean stddev n'
cmp='diff -cb '
