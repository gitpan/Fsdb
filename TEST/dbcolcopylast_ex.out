#fsdb experiment mean stddev n copylast_experiment copylast_mean copylast_stddev copylast_n
ufs_mab_sys	37.25	0.070711	2	-	-	-	-
ufs_rcp_real	271.2	9.4752	2	ufs_mab_sys	37.25	0.070711	2
#  | dbmultistats -k experiment duration
#   | dbcol experiment mean stddev n
#   | dbcolcopylast experiment mean stddev n
